cluster/addons/kube-network-policies/kube-network-policies.yaml

```diff
@@ -27,7 +27,7 @@ spec:
       serviceAccountName: kube-network-policies
       containers:
       - name: kube-network-policies
-        image: registry.k8s.io/networking/kube-network-policies:v0.1.0
+        image: gcr.io/k8s-staging-networking/kube-network-policies:v20240423-88bdf40
```

Comment:
can we publish an image?

we generally want to be pulling through registry.k8s.io

Reply:
(and we generally want k/k on release tags)

Reply:
I wanted to get the CI green before cut another release, once the CI is stable I'll switch again to registry